dstep/driver/Application.d

```diff
@@ -37,20 +38,24 @@ class Application : DStack.Application
 
     private
     {
-        Language language;
+        Options defaultOptions;
```

Comment:
Yeah, I don't know which one is best. `defaultOptions` made more sense when I though it was constant.